src/wp-admin/edit-form-blocks.php

```diff
@@ -77,16 +77,22 @@ static function( $classes ) {
  * Assign initial edits, if applicable. These are not initially assigned to the persisted post,
  * but should be included in its save payload.
  */
-$initial_edits = null;
+$initial_edits = array();
 $is_new_post   = false;
 if ( 'auto-draft' === $post->post_status ) {
 	$is_new_post = true;
 	// Override "(Auto Draft)" new post default title with empty string, or filtered value.
-	$initial_edits = array(
-		'title'   => $post->post_title,
-		'content' => $post->post_content,
-		'excerpt' => $post->post_excerpt,
-	);
+	if ( post_type_supports( $post->post_type, 'title' ) ) {
+		$initial_edits['title'] = $post->post_title;
+	}
+
+	if ( post_type_supports( $post->post_type, 'content' ) ) {
```

Comment:
@hectorfarahani @audrasjb I think it should be 'editor' instead of 'content'? Posts don't have a 'content' supports property

There's a related e2e test now failing in gutenberg:
```
new editor filtered state › should respect default content

    expect(received).toBe(expected) // Object.is equality

    Expected: "My default content"
    Received: ""
```

Reply:
Put together a fix in https://github.com/WordPress/wordpress-develop/pull/1929

Reply:
@talldan My bad, I see it is fixed on https://github.com/WordPress/wordpress-develop/pull/1929
Does any action needed from my side?